media_driver/agnostic/common/os/mos_utilities.c

```diff
@@ -6658,3 +6658,1008 @@ void Mos_SwizzleData(
         }
     }
 }
+
+//*********This is ISL (Intel Surface Layout) implementation for sw-based tiling/detiling*********
+
+#define INLINE_SSE41
```

Comment:
I thought on mesa this was in a separate file. Can this be done the same way here? It will be easier to compare against mesa, hence support.